src/components/Envoy/tables/ClusterTable.ts

```diff
@@ -1,24 +1,24 @@
-import { defaultFilter, SummaryTable, SummaryTableRenderer } from './BaseTable';
+import { SummaryTable, SummaryTableRenderer } from './BaseTable';
 import { ICell, ISortBy, sortable, SortByDirection } from '@patternfly/react-table';
 import { ClusterSummary } from '../../../types/IstioObjects';
-import { FILTER_ACTION_APPEND, FilterType, FilterTypes } from '../../../types/Filters';
-
-const filterToColumn = {
-  fqdn: 0,
-  port: 1,
-  subset: 2,
-  direction: 3
-};
+import { ActiveFilter, FILTER_ACTION_APPEND, FilterType, FilterTypes } from '../../../types/Filters';
+import { SortField } from '../../../types/SortFilters';
+import Namespace from '../../../types/Namespace';
+import { defaultFilter, istioConfigLink, serviceLink } from '../../../helpers/EnvoyHelpers';
 
 export class ClusterTable implements SummaryTable {
   summaries: ClusterSummary[];
   sortingIndex: number;
   sortingDirection: 'asc' | 'desc';
+  namespaces: Namespace[] | undefined;
+  namespace: string;
 
-  constructor(summaries: ClusterSummary[], sortBy: ISortBy) {
+  constructor(summaries: ClusterSummary[], sortBy: ISortBy, namespaces: Namespace[] | undefined, namespace: string) {
```

Comment:
Not sure if that "undefined" is necessary or if there is a case where namespace array is undefined instead of an empty one.

Anyway, it's minor.

I saw that in other places as well.

Reply:
Approached